libs/edu/ucla/library/vertx-auth-ldap/4.4.0/vertx-auth-ldap-4.4.0.jar.md5

```diff
@@ -0,0 +1 @@
+b983f3670f6f4376b542c60f340856a9
```

Comment:
Should the md5/sha1 files be gitignored? If not, why/how are they necessary to building/deploying the app?

Reply:
This is a tiny 3rd party Maven repository within this project's directory (the repo directory structure mirrors a real Maven repo). The ldap dependency is "downloaded" from this repo and the MD5 file serves as the check that the Maven process has to confirm the file was "downloaded" successfully.

It might work without the checkfile (I think probably it would(?)), but I didn't check to see if it would output a warning or anything since they don't hurt (I don't think?), and they are temporary. When the vertx-ldap project is set up in its one project, it's artifact will be published to Maven Central and then this project can be updated to just pull from there instead.